tests/integration/ha_test.go

```diff
@@ -124,18 +124,21 @@ func TestInitWithMultipleKeepers(t *testing.T) {
 }
 
 func setupServers(t *testing.T, clusterName, dir string, numKeepers, numSentinels uint8, syncRepl bool, usePgrewind bool, primaryKeeper *TestKeeper) (testKeepers, testSentinels, *TestProxy, *TestStore) {
-	var initialClusterSpec *cluster.ClusterSpec
+	var initialClusterSpec = &cluster.ClusterSpec{
+		SleepInterval:          &cluster.Duration{Duration: 2 * time.Second},
+		FailInterval:           &cluster.Duration{Duration: 5 * time.Second},
+		ConvergenceTimeout:     &cluster.Duration{Duration: 30 * time.Second},
+		MaxStandbyLag:          cluster.Uint32P(50 * 1024), // limit lag to 50kiB
+		SynchronousReplication: cluster.BoolP(syncRepl),
+		PGParameters:           defaultPGParameters,
+		// If we want to pg_rewind then also checkpoint beforehand, otherwise we may fail due
+		// to uncheckpoint'ed timeline changes that have us silently fallback to basebackup.
+		UsePgrewind:              cluster.BoolP(usePgrewind),
+		CheckpointBeforePgrewind: cluster.BoolP(usePgrewind),
```

Comment:
This is open for questioning: should we assign this by default? It doesn't harm us now but we may want to test when the parameter is *not* set (though I think this is unlikely).

Perhaps ok for now, until we have an issue?